### PR TITLE
fix: half all traffic for lossy

### PR DESCRIPTION
**Pull request by auspham**
### Description of PR

Halving the traffic on test and background flow for compatibility between 400g and 200g for lossy

Summary:
Fixes # (issue) 30112807

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ ] 202405

### CODE REVIEW

File: tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py [hunk 1]
```diff
@@ -67,6 +67,11 @@ def test_pfc_pause_single_lossy_prio(snappi_api,                # noqa: F811
     snappi_extra_params = SnappiTestParams()
     snappi_extra_params.multi_dut_params.multi_dut_ports = snappi_ports
 
+    flow_factor = 1
+
+    if snappi_extra_params.multi_dut_params.multi_dut_ports[0]['duthost'].facts()['asic_type'] == 'cisco-8800':
```
Comment (sdszhang) [inline tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py:72]: can we also check if port speed is 400G?

Comment (auspham) [reply tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py]: updated

**Comment by sdszhang** [comment]
@zhixzhu @rraghav-cisco to review.
@abdosi @alpeshspatel for viz.

File: tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py [hunk 2]
```diff
@@ -67,6 +67,11 @@ def test_pfc_pause_single_lossy_prio(snappi_api,                # noqa: F811
     snappi_extra_params = SnappiTestParams()
     snappi_extra_params.multi_dut_params.multi_dut_ports = snappi_ports
 
+    flow_factor = 1
+
+    if snappi_ports[0]['duthost'].facts()['asic_type'] == 'cisco-8800' and snappi_ports[0]['speed'] == '400000':
+        flow_factor = 2
```
Comment (rraghav-cisco) [inline tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py:73]: We can use 
if snappi_ports[0]['duthost'].facts()['asic_type'] == 'cisco-8800':
    if int(snappi_ports[0]['speed']) > 200000:
        flow_factor = int(snappi_ports[0]['speed']) / 200000.0

To account for possible future 800G speeds. @sdszhang, @abdosi : pls comment.

Comment (auspham) [reply tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py]: @rraghav-cisco  i have updated the logic

**Review by rraghav-cisco — APPROVED** [review]

**Review by sdszhang — APPROVED** [review]

**Comment by auspham** [comment]
tested 3 times all passed

**Review by abdosi — APPROVED** [review]

**Pull request merged**